packages/lit/src/utils/range-synchronizer.ts

```diff
@@ -140,7 +140,9 @@ export class RangeSynchronizer {
     if (!range) return;
 
     const blocks = this._rangeManager.getSelectedBlockElementsByRange(range, {
-      match: element => element.model.role === 'content',
+      match: element =>
+        element.model.role === 'content' ||
+        element.model.flavour === 'affine:surface-ref',
```

Comment:
We shouldn't sync this block for native selection. Consider handle the edge case in `Delete` keymap.